scrapers/ar/bills.py

```diff
@@ -215,6 +206,22 @@ def scrape_actions(self):
                 action, date, chamber=actor, classification=action_type
             )
 
+    def get_entity_name(self, link):
+        entity_type = "person"
+        if "Committees" in link:
+            entity_type = "organization"
+        return entity_type
+
+    def get_chamber(self, name):
+        chamber = ""
+        if "senator" in name.lower():
+            chamber = "upper"
+        elif "representative" in name.lower():
+            chamber = "lower"
+        elif "joint" in name.lower():
+            chamber = "legislature"
+        return chamber
```

Comment:
I found a couple data points where we are missing a `chamber` value, both are organization sponsorships:

```
jesse@greenbookwork:~/repo/openstates/braykuka/openstates-scrapers/scrapers/_data/ar$ cat bill_3775aae8-7d2b-11ef-ad10-4c034f4dc15f.json | jq .sponsorships
[
  {
    "name": "Senate Efficiency",
    "classification": "primary",
    "entity_type": "organization",
    "primary": true,
    "person_id": null,
    "organization_id": "~{\"chamber\": \"\", \"name\": \"Senate Efficiency\"}"
  }
]
```

```
jesse@greenbookwork:~/repo/openstates/braykuka/openstates-scrapers/scrapers/_data/ar$ cat bill_cf7f310a-7d2c-11ef-ad10-4c034f4dc15f.json | jq .sponsorships
[
  {
    "name": "House Management",
    "classification": "primary",
    "entity_type": "organization",
    "primary": true,
    "person_id": null,
    "organization_id": "~{\"chamber\": \"\", \"name\": \"House Management\"}"
  }
]
```

In these cases the `entity_type` is correct, and it looks like the could could be expanded to identify that "House Management" is a `lower` chamber organization, and `Senate Efficiency` is an `upper` chamber organization (house=lower, senate=upper). Maybe just additional `elif` cases after line 222 for `"house" in name.lower()` and `"senate" in name.lower()`?